src/OrbitQt/CallTreeWidget.cpp

```diff
@@ -919,7 +919,7 @@ void CallTreeWidget::ProgressBarItemDelegate::paint(QPainter* painter,
   }
 
   bool highlight = index.data(CallTreeViewItemModel::kMatchesCustomFilterRole).toBool();
-  if (option.state & QStyle::State_Selected) {
+  if (option.state & QStyle::State_Selected != 0u) {
```

Comment:
Doesn't `!=` have higher precedence than `&`? We need parentheses here.

Reply:
Oh nice, this actually triggers a compiler error

Reply:
Thanks for the catch (while I am happy that the compiler also got this). The changes were autogenerated by clang-tidy, and I missed that when proofreading them.

Reply:
Done.